AUTHENTICATION.md

```diff
@@ -5,19 +5,93 @@ The recommended way to authenticate to the Google Cloud PHP library is to use
 which discovers your credentials automatically, based on the environment where your code is running.
 To review all of your authentication options, see Project and Credential lookup.
 
-For more information about authentication at Google, see [the authentication guide](https://cloud.google.com/docs/authentication). 
-Specific instructions and environment variables for each individual service are linked from the README documents listed below for each service.
+## Quickstart
+
+Here is a quick an opinionated quickstart guide that serves as a means to get started as quick as possible. There are different ways to authenticate
+a client that we will go over this guide, but we wanted to offer a quick way to configure the authentication. We will go over the steps of what needs to be done in
+order to authenticate a client first and then we will explain said steps in detail for a quickstart.
+
+### For local development:
+* Install the Google Cloud CLI.
+* Authenticate with `gcloud` to generate the credentials file.
+* Instantiate a client.
+
+### Installing the Google Cloud CLI
+In order to generate our needed credentials file we need to authenticate to gcloud first. Installation is handled differently depending on your platform.
+Here is a link to help you setup the Google Cloud CLI:
+
+https://cloud.google.com/sdk/docs/install
+
+### Authenticate via the `gcloud` command
+Once the Google Cloud CLI tools are installed it is required that we authenticate via the `gcloud`:
+```shell
+$ gcloud init
+$ gcloud auth application-default login
+```
 
-## Project and Credential Lookup
+This will create a local file in your system that the authentication library for our client will read in order to make requests to the apis with those credentials.
+This file is located in different place depending on your system.
 
-The Google Cloud PHP library provides several mechanisms to configure your system without 
-providing **Project ID** and **Service Account Credentials** directly in code.
+Windows:
+```
+%APPDATA%\gcloud\application_default_credentials.json
+```
+
+Linux and MacOS:
+```
+$HOME/.config/gcloud/application_default_credentials.json
+```
+
+### Instantiating the client
+Once we have the `application_default_credentials.json` that we created on the previous step now we can instantiate a client which internally using the Google Auth library
+will take that file and use it to authenticate your requests:
+```php
+use Google\Cloud\Translate\V3\Client\TranslationServiceClient;
+use Google\Cloud\Translate\V3\TranslateTextRequest;
 
-**Project ID** is discovered in the following order:
+// Instantiating the client gathers the credentials from the `application_default_credentials.json`
+$client = new TranslationServiceClient();
```

Comment:
I think it'd be useful to have _somewhere_ how they install this library. It can just be something like:

```
**Note**: You can install the Google Translate client library by running `composer install google/cloud-translate`.
See the [README.md][README.md] for further instructions, or browse [the complete list of packages](https://cloud.google.com/php/docs/reference/).
```